C backend compile error under Windows (msys2)

A file test.fz containing:

HelloWorld is
  say "Hello World!"

gives the following error when compiled with the C backend on Windows in a MSYS shell:

ivoba@DESKTOP-KN14QQK MSYS /d/fuzion/tests
$ fz -c test.fz

warning 1: Platform encoding is windows-1252, source files are nevertheless treated as UTF8 files.

**HelloWorld.c:3:10: fatal error: 'unistd.h' file not found
#include <unistd.h>
         ^~~~~~~~~~
1 error generated.**

error 1: C backend: C compiler failed
C compiler call 'clang -O3 -o HelloWorld HelloWorld.c' failed with exit code '1'

one error and one warning.

I found some more info on this here:

[stackoverflow](https://stackoverflow.com/questions/341817/is-there-a-replacement-for-unistd-h-for-windows-visual-c)

windows.h is suggested as a replacement for this, it can be found at C:\tools\msys64\mingw64\x86_64-w64-mingw32\include

--- a/README.md
+++ b/README.md
@@ -23,7 +23,7 @@ Please check [https://flang.dev](https://flang.dev) for language and implementat
 2) In Powershell:
 2.1) choco install git openjdk make msys2 diffutils
 2.2) [Environment]::SetEnvironmentVariable("Path","c:\tools\msys64\mingw64\bin;" + $env:Path , "User")
-3) In File C:\tools\msys64\msys2_shell.cmd change line: 'rem setMSYS2_PATH_TYPE=inherit' to 'set MSYS2_PATH_TYPE=inherit'
+3) In file C:\tools\msys64\msys2_shell.cmd change line: 'rem set MSYS2_PATH_TYPE=inherit' to 'set MSYS2_PATH_TYPE=inherit'
 4) In msys2 shell (execute C:\tools\msys64\msys2_shell.cmd):
 4.1) pacman -S mingw-w64-x86_64-clang
 4.2) make

--- a/README.md
+++ b/README.md
@@ -20,10 +20,13 @@ Please check [https://flang.dev](https://flang.dev) for language and implementat
 > Note that building from powershell/cmd does not work yet.
 
 1) Install chocolatey: https://chocolatey.org/install
-2) choco install git openjdk make msys2 diffutils
-3) in C:\tools\msys64\msys2_shell.cmd change line: 'rem set MSYS2_PATH_TYPE=inherit' to 'set MSYS2_PATH_TYPE=inherit'
-4) execute: C:\tools\msys64\msys2_shell.cmd
-5) pacman -S mingw-w64-x86_64-clang
+2) In Powershell:
+2.1) choco install git openjdk make msys2 diffutils
+2.2) [Environment]::SetEnvironmentVariable("Path","c:\tools\msys64\mingw64\bin;" + $env:Path , "User")
+3) In File C:\tools\msys64\msys2_shell.cmd change line: 'rem setMSYS2_PATH_TYPE=inherit' to 'set MSYS2_PATH_TYPE=inherit'
+4) In msys2 shell (execute C:\tools\msys64\msys2_shell.cmd):
+4.1) pacman -S mingw-w64-x86_64-clang
+4.2) make
 
 ## Build
 

